### PR TITLE
Initial Setup: Create Branch, .gitignore, Linter, README, and Files

### PR DESCRIPTION
**Pull request by ahmidou09**
This pull request includes the initial setup steps for the project, Here is a breakdown of the changes made in this PR:

- Created a new GitHub repository
- Creating a new branch for development
- Added a .gitignore
- Included "node_modules" in the gitignore file to exclude it from being tracked.
- Set up a linter for HTML and CSS
- Committed three files in separate commits:
   1. HTML file: Added an h1 header with the text "Hello Microverse!"
   2. CSS file: Added a class with styles to customize the header
   3. Test.md file: Created a file with the text "This file should be ignored by git"

### CODE REVIEW

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @ahmidou09         , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Linters are passing 👍 




 

## Required Changes ♻️

- [ ] kindly consider adding a descriptive title on your pr (a descriptive title talks about the changes introduced in the current pr) after that consider also adding a descriptive pr description(pr description talks in detail about what the branch with the pr contains) doing so will make your pr more descriptive and professional
- [ ] kindly consider removing the Microverse logo or add yours if you've got one as this repo is not microverse's this will make your readme customized with your own data
- [ ] kindly consider customizing your readme by making each section contain information related to this current project you are working on and while doing so consider removing all those grey comments and implementing what they talk about as they were given as guidance to guide you not to be included in your readme doing so will make your readme look professional and more descriptive

- [ ] kindly next time consider using descriptive commit messages by making the in imperative form to make the clear related to what they contain

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -58,231 +22,69 @@ After you're finished please remove all the comments and instructions!
...
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- ACKNOWLEDGEMENTS -->
-
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+-
```
Comment (AbelG101) [inline README.md]: - [ ] It seems like there's been some confusion as the reviewer previously mentioned you should only remove the section descriptions (the gray comments) from the readme but below each heading you should add descriptive information and you shouldn't leave it empty. for example in the **key features** heading you should add at least 1 feature of the project and the same applies to the rest of the sections as well. 😉

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @ahmidou09 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.

## Optional suggestions
- [OPTIONAL] I'd recommend you name your branch with the name of a feature or action (e.g. 'created_hello_microverse' or 'added_initial_structure') 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @ahmidou09         , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Linters are passing 👍 




 

## Required Changes ♻️

- [ ] kindly while implementing the requested change in the previous review process kindly consider making sure that you don't remove some sections in your readme by checking that you follow exactly the template provided cause currently you deleted the future feature and  and the getting started ,.....remember that The only optional section is FAQ.this will make you your readme more descriptive

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmidou09** [comment]
Ok I will do that thanks for your efforts

**Review by topeogunleye — APPROVED** [review]
### Hi @ahmidou09, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Added back the future features and getting started sections to the README file as suggested in the previous review. ✔️ 
 
## [Optional] suggestions:
* [ ] For future pull requests, please notice that your branch name `first-branch` does not follow the convention of being named after a feature or action. For example, you could name your branch `added-html-css`. This makes it easier to track the changes and the purpose of each branch..
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmidou09** [comment]
Thank you all

**Pull request merged**